hpOneView/resources/resource.py

```diff
@@ -74,14 +75,22 @@ class Resource(object):
     """
     This class implements common functions for HpOneView API rest
     """
-
+    # Base URI for the rest calls
     URI = '/rest'
+
+    # Unique identifiers to query the resource
+    UNIQUE_IDENTIFIERS = ['uri', 'name']
+
+    # Add fields to be removed from the request body of a post request
+    EXCLUDE_FROM_REQUEST = []
+
+    # Default values required for the api versions
     DEFAULT_VALUES = {}
 
-    def __init__(self, connection, data):
+    def __init__(self, connection, data=None):
 
         self._connection = connection
-        self.data = data
+        self.data = data if data else {}
```

Comment:
default value itself can be changed

Reply:
We should not keep the mutable object as a default value.